Training 453/app/js/vendor/modernizr-2.6.2-respond-1.1.0.min.js

```diff
@@ -0,0 +1,11 @@
+/* Modernizr 2.6.2 (Custom Build) | MIT & BSD
```

Comment:
this file can be moved as a node_module in package json instead of copying it here